### ISSUE TITLE
Fix 'Nan' error while training a brain

### ISSUE THREAD

**Issue by StepNeverStop** [Unity-Technologies/ml-agents#1808]
when training my agent by using my own algorithm with the official conda environment "mlunity", i found every time when i trained for about 200 episodes there always some 'nan' values be generated.
i try to fix this problem for about half a day, and i tried to adjust the learning rate, network structure ... 
anything that i have tried didn't work.
finally , i created my own conda environment with tensorflow or tensorflow-gpu of version 1.12.1 , not the version of 1.7.1 in official conda env.
It worked. but i still dont know why , why it happened, and why doing that could solver this problem by using another version of tensorflow which is different with the offical one. But, It worked.
I guess maybe there is some wrong with the gradient spread about the network.
I strongly suggest the official could upgrate their tensorflow VERSION in conda environment 'mlunity' to aviod some unkown error that may happen.
sry for my bad ENG grammer.

**Comment by ervteng** [comment]
Hi @StepNeverStop, thanks for the heads-up! We don't have an official conda environment called `mlunity` - where did you find this environment? What the guide is doing is simply creating a virtual environment so you can install your own packages. 

We'll look into upgrading the TF version, too. As far as training is concerned, you should be able to use any version newer than 1.7 - I've tested it up to the current newest version with no issue. You may have issues with inference, however, as that is untested.

**Comment by xiaomaogy** [comment]
Thanks for reaching out to us. Hopefully you were able to resolve your issue.  We are closing this due to inactivity, but if you need additional assistance, feel free to reopen the issue.
